src/BenchmarksClient/Startup.cs

```diff
@@ -163,7 +163,7 @@ private static Process StartProcess(ClientJob job)
                 }
             }
 
-            command += $" --latency -d {job.Duration} -c {job.Connections} --timeout 8 -t {job.Threads}  {job.ServerBenchmarkUri}";
+            command += $" --latency -d {job.Duration} -c {job.Connections} --timeout 8 -t {job.Threads}  {job.ServerBenchmarkUri}{job.Query}";
```

Comment:
Is this missing a space before `{job.Query}`?

Reply:
No.  `{job.Query}` is a query string, like `?queries=20`, so it needs to be appended to the URL without a space.